doc/getting-started/getting-started.rst

```diff
@@ -515,7 +567,7 @@ Build ACRN
 
       cd ./build
       ls *.deb
-         acrn-my_board-shared-2.7.deb
+         acrn-my_board-shared-2.7.deb # TODO update file name
```

Comment:
Maybe we should change these references now to be ``acrn-my_board-shared-*.deb`` ?  Does that match the file names you're seeing for the hypervisor build output?

Reply:
Unfortunately, I haven't been able to build successfully, so I'm not sure what the file name is. I filed a ticket in JIRA with the error details.

Reply:
In that case, let's just go with it as it is and fix later.